backend/src/test/java/hanglog/trip/presentation/TripControllerTest.java

```diff
@@ -53,7 +58,19 @@ void createTrip() throws Exception {
                         .contentType(APPLICATION_JSON)
                         .content(objectMapper.writeValueAsString(tripRequest)))
                 .andExpect(status().isCreated())
-                .andExpect(header().string(LOCATION, "/trips/1"));
+                .andExpect(header().string(LOCATION, "/trips/1"))
+                .andDo(
+                        restDocs.document(
+                                requestFields(
+                                        fieldWithPath("startDate").type(JsonFieldType.STRING).description("여행 시작 날짜").attributes(field("constraint", "yyyy-MM-dd")),
+                                        fieldWithPath("endDate").type(JsonFieldType.STRING).description("여행 종료 날짜").attributes(field("constraint", "yyyy-MM-dd")),
+                                        fieldWithPath("cityIds").type(JsonFieldType.ARRAY).description("도시 ID 목록").attributes(field("constraint", "1개 이상의 양의 정수"))
```

Comment:
```suggestion
requestFields(
                                        fieldWithPath("startDate").type(JsonFieldType.STRING).description("여행 시작 날짜")
                                                .attributes(field("constraint", "yyyy-MM-dd")),
                                        fieldWithPath("endDate").type(JsonFieldType.STRING).description("여행 종료 날짜")
                                                .attributes(field("constraint", "yyyy-MM-dd")),
                                        fieldWithPath("cityIds").type(JsonFieldType.ARRAY).description("도시 ID 목록")
                                                .attributes(field("constraint", "1개 이상의 양의 정수"))
                                ),
```

여기도 가로줄 제한을 넘겨서 정렬하면 이렇게 바뀌는데 type과 description도 줄바꿈 하는 방법도 좋아 보입니다.!

Reply:
적용하였습니다 :)